snippets/footer.html

```diff
@@ -0,0 +1,8 @@
+<div class="row footer">
+    <div class="col-lg-4"></div>
+    <div class="col-lg-4 text-center">
+      <p><a href="../icon-credit-source.html" target="_blank">Credit given for icons used</a> </p>
```

Comment:
please use absolute path from project root so that we don't have to change multiple things if files move around:

```html
<a href="/icon-credit-source.html" target="_blank">
```